src/datasets/ping_intro.md

```diff
@@ -87,14 +87,15 @@ contains a Telemetry Environment, [Crash Annotations][crash_annotations], and
 
 It was introduced in Firefox 40.
 
-### "optout" ping
+### "deletion request" ping
 
 In the event a user opts out of Telemetry, we send one final
-["optout" ping][optout_ping] to let us know. We try exactly once to send it,
-discarding the ping if sending fails. It contains only the
-[common ping data][common_ping_data] and an empty payload.
+["deletion-request" ping][deletion_request_ping] to let us know. We try
+exactly once to send it, discarding the ping if sending fails. It contains
+only the [common ping data][common_ping_data] and an empty payload.
 
-It was introduced in Firefox 63.
+It was introduced in Firefox 70, replacing the ["optout" ping][optout_ping]
```

Comment:
72, actually.

Reply:
https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/data/deletion-request-ping.html implies 70, is it wrong?

Reply:
It is! (I'll file a bug) (I filed it: https://bugzilla.mozilla.org/show_bug.cgi?id=1598447 )